lib/run_qatr.pm

```diff
@@ -0,0 +1,24 @@
+use base "opensusebasetest";
+use testapi;
+use strict;
+use utils;
+
+
+sub run {
+	my ($self,$test) = @_;
+	$self->select_serial_terminal;
+
+	my $HOST       = get_required_var('HOST_IP');
+	my $USER       = get_required_var('HOST_USER');
+	my $TESTS_PATH = get_required_var('TESTS_PATH');
+	my $IMAGE      = get_required_var('IMAGE');
+
+	my $path = $TESTS_PATH . $test;
+	my $image = '/var/lib/openqa/share/factory/hdd/' . $IMAGE;
+	my $cmd='qatrfm ' . ' --path ' . $path . ' --image ' . $image;
```

Comment:
different tests might require different input variables (e.g. >1 num_domains).. not sure how to introduce it here, maybe as arguments from main?